### PR TITLE
Added authz support for mgmt

### PR DESCRIPTION
**Pull request by slavikm**
## Description
Support ReBAC authorization capabilities in the management SDK

## Must
- [ ] Tests
- [ ] Documentation (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ These sections show how to use the SDK to perform API management functions. Befo
 9. [Manage JWTs](#manage-jwts)
 10. [Search Audit](#search-audit)
 11. [Embedded Links](#embedded-links)
+12. [Manage Authz](#manage-authz)
```
Comment (aviadl) [inline README.md:65]: Do we want to add the term ReBAC to menu ?

File: descope/sdk/mgmt.go [hunk 2]
```diff
@@ -454,6 +454,63 @@ type Audit interface {
...
+	// DeleteSchema for the project which will also delete all relations.
+	DeleteSchema() error
+
+	// LoadSchema for the project.
+	LoadSchema() (*descope.AuthzSchema, error)
+
+	// SaveNamespace creating or updating the given namespace
+	// Will not delete relation definitions not mentioned in the namespace.
+	// oldName is used if we are changing the namespace name
+	// schemaName is optional and used to track the current schema version.
```
Comment (aviadl) [inline descope/sdk/mgmt.go:473]: and can be used to track

File: examples/managementcli/main.go [hunk 3]
```diff
@@ -620,6 +647,13 @@ func main() {
 		cmd.Args = cobra.ExactArgs(2)
 	})
 
+	addCommand(authzLoadSchema, "authz-load-schema", "Load and display the current AuthZ ReBAC schema", func(cmd *cobra.Command) {
```
Comment (asafshen) [inline examples/managementcli/main.go:650]: out of curiosity - why only those two? (I'm not arguing we should add all of the authz api here, just wondering what is the logic behind those two, if we want to tell the entire story, maybe also expose save schema and add a relation - like you mentioned in the readme)

**Review by asafshen — APPROVED** [review]
cool(z) :)

**Review by aviadl — APPROVED** [review]
But please see think about my docs comments

File: examples/managementcli/main.go (hunk 3 above)
Comment (slavikm) [reply examples/managementcli/main.go:650]: Just me being lazy. I'll add the save schema and add relation as well so you can have the full cycle via CLI.

File: README.md (hunk 1 above)
Comment (slavikm) [reply README.md:65]: Sure

**Review by asafshen — APPROVED** [review]

**Pull request merged**